### PR TITLE
Integration

### PR DESCRIPTION
**Pull request by jakeware**
Read imu and camera strobe into circular buffers and periodically push that data into a send buffer for usb.

### CODE REVIEW

File: src/svis_teensy.cpp [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+}
+
+void InitInterrupts() {
+  // setup interrupt timers
+  imu_timer.begin(ReadIMU, 1000);  // microseconds
+  imu_timer.priority(0);  // [0,255] with 0 as highest
+
+  // setup pin interrupt
+  // TODO(jakeware): What is the priority of this?
+  attachInterrupt(7, ReadStrobe, RISING);  // attach pin 7 to interrupt
```
Comment (jeswiezy) [inline src/svis_teensy.cpp]: Does this mean we're having the i7 control the camera and we just read the strobe when it happens? What if the i7 boots up before the Teensy? Would the counts be off?

File: src/svis_teensy.cpp [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+}
+
+void InitGPIO() {
+  // configure onboard LED
+  pinMode(LED_PIN, OUTPUT);
+}
+
+void InitInterrupts() {
+  // setup interrupt timers
+  imu_timer.begin(ReadIMU, 1000);  // microseconds
```
Comment (jeswiezy) [inline src/svis_teensy.cpp]: Why are we using a timer to poll the IMU when we have an interrupt line that can tell us when new data is ready?

File: src/svis_teensy.cpp (hunk 1 above)
Comment (jakeware) [inline src/svis_teensy.cpp]: We are currently doing that, but we can always change it later.  I think the two situations are equivalent as far as the uncertainty around missing an image or strobe message.  No matter how we do it, the counts are going to be off at first and will have to be synced on the i7.  Still working out the best way to do this.

File: src/svis_teensy.cpp (hunk 2 above)
Comment (jakeware) [inline src/svis_teensy.cpp]: Good point.  I actually didn't realize this existed.  We should transition to using it.  I created an issue for it in #11.

**Pull request merged**